packages/shared/src/components/widgets/ReferralSocialShareButtons.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactElement } from 'react';
+import { Button } from '../buttons/Button';
+import {
+  getFacebookShareLink,
+  getTwitterShareLink,
+  getWhatsappShareLink,
+} from '../../lib/share';
+import WhatsappIcon from '../icons/Whatsapp';
+import FacebookIcon from '../icons/Facebook';
+import TwitterIcon from '../icons/Twitter';
+import { link } from '../../lib/links';
+import { labels } from '../../lib';
+import { SimpleTooltip } from '../tooltips';
+
+const ReferralSocialShareButtons = ({ url }: { url: string }): ReactElement => {
```

Comment:
I think I didn't add this yet, but this should have tracking @sshanzel not sure you wanted to add it seperate?

Reply:
I think there is nothing left for us, so I thought of having the analytics here too.

Reply:
Yeah sure can add that here if we want

Reply:
Just pushed the events as well now.